src/utils/utils_sanitizers.h

```diff
@@ -168,6 +168,24 @@ static inline void utils_annotate_memory_inaccessible(void *ptr, size_t size) {
 #endif
 }
 
+static inline void utils_annotate_memory_new(void *ptr, size_t size) {
+#ifdef UMF_VG_MEMCHECK_ENABLED
```

Comment:
Should it be `UMF_VG_DRD_ENABLED`?
The `ANNOTATE_NEW_MEMORY` is defined in `drd.h` header.

Were you able to compile this code?

Reply:
fixed